src/Sulu/Bundle/MediaBundle/Markup/MediaTag.php

```diff
@@ -144,9 +152,46 @@ function ($attributes) {
         $result = [];
         foreach ($medias as $media) {
             $media['url'] = $this->mediaManager->getUrl($media['id'], $media['name'], $media['version']);
+            $dispositionType = $this->findMediaDispositionType($media);
+
+            if ($dispositionType) {
+                $media['url'] .= '&' . $dispositionType . '=true';
+            }
+
             $result[$media['id']] = $media;
         }
 
         return $result;
     }
+
+    /**
+     * Find disposition type by media MIME type.
+     *
+     * @param array $media
+     *
+     * @return string|null
+     */
+    private function findMediaDispositionType(array $media)
+    {
+        if (
+            !isset($media['mimeType']) ||
+            !$media['mimeType'] ||
+            !isset($this->dispositionType['default']) ||
+            !$this->dispositionType['default']
```

Comment:
Shouldn't this function work even if there is no `default` key? I think I would remove the previous two lines. But maybe I just don't understand something.